guides/manual-installation.md

```diff
@@ -479,54 +468,43 @@ e859c07888ca   registry.suse.com/trento/trento-wanda:1.2.0   "/bin/sh -c '/app/b
 
 Both containers should be running and listening on the specified ports.
 
-### Setup nginx as reverse proxy
-
-#### Option 1: Creating a Self-Signed Certificate
-
-> Note: This is a basic guide for creating a self-signed certificate for use with Trento. You may use your own certificate. For detailed instructions, consult the OpenSSL documentation. Remember to replace example.com with your domain.
+### Validate the health status of trento web and wanda
+Validate the health status of the Trento web and wanda services locally by accessing the ```healthz``` and ```readyz``` api.
```

Comment:
I would put `Trento web and wanda services correct functioning could be validated accessing the...`